share/spice/just_delete_me/content.handlebars

```diff
@@ -3,13 +3,8 @@
     {{notes}}
   </p>
 {{/if}}
-{{#if delete_url}}
```

Comment:
I'm not quite certain what you mean. As a tile, clicking on the tile will go to the url, and on detail (single item view) clicking the title (e.g. Facebook) go to the url.